.github/workflows/build-test-docs.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and test documentation
+
+on:
+  push:
+    # all branches
+    paths:
+      - 'documentation/**/*.rst'
+      - 'documentation/**/conf.py'
+
+  pull_request:
+    # all branches
+    paths:
+      - 'documentation/**/*.rst'
+      - 'documentation/**/conf.py'
+
+  # This enables the Run Workflow button on the Actions tab.
+  workflow_dispatch:
+
+jobs:
+  build-documentation:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Link check
+        uses: addnab/docker-run-action@v3
+        with:
+          image: ghcr.io/fraya/dylan-docs
```

Comment:
Do you have notes somewhere that describe how this image was made? Perhaps link to them in a comment. (optional)

Reply:
In the [repository of dylan-docs](https://github.com/fraya/dylan-docs) there is the [Dockerfile](https://github.com/fraya/dylan-docs/blob/main/Dockerfile) that creates the image and the [GH Action](https://github.com/fraya/dylan-docs/blob/main/.github/workflows/create-and-publish-docker-image.yml) shows the download of the sphinx-contrib package and the push of the image. If you like I can add this comment.